examples/vmseries/example.tfvars

```
@@ -0,0 +1,8 @@
+location            = "Australia East"
+resource_group_name = "example-rg"
+common_vmseries_sku = "bundle1"
+username            = "panadmin"
+allow_inbound_mgmt_ips = [
+  "191.191.191.191", # Put your own public IP address here
```

```suggestion
  "191.191.191.191", # Put your own public IP address here, visit "https://ifconfig.me/"
```
nit: Since it's an educational example, maybe let's give a hint for the user how to quickly check the public IP address?